src/features/profile/components/properties/Properties.tsx

```diff
@@ -0,0 +1,39 @@
+import Property from './Property';
+import './properties.scss';
+
+type CommonProperties = Record<
+  | 'electricity'
+  | 'gate'
+  | 'lighting'
+  | 'wasteCollection'
+  | 'water'
+  | 'summerStorageForDockingEquipment'
+  | 'summerStorageForTrailers',
+  boolean
+>;
```

Comment:
This should probably be called `AllProperties`, as all the properties are not common between the two place types

Reply:
Fair enough, renamed! 😅